tools/aws_lab_setup/roles/reachability/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: Wait 400 seconds, but only start checking after 30 seconds
```

Comment:
Please get rid of this role. It should be task of the instance provisioning automation.

Reply:
@tima can we leave it as a role for now, or change it from a role into a task within the playbook, it has to be another play (see bug:  https://github.com/ansible/ansible/issues/35195) . the only way to keep it within the same role is to use this meta module which doesn't seem to be working.

Reply:
OK. Yeah that's out of your control. It doesn't make me happy, but that's not on you. Would appreciate a brief comment with a link reminding us of that bug in there for the next person who wonders why this role exists.

Would you consider a different name along the lines of "connectivity test" or the like?